### PR TITLE
Add zulip link to PR template

### PR DESCRIPTION
**Pull request by lucyleeow**
# Fixes/Closes

# Description

Adds link to review channel in zulip, so contributors know that they can message for reviews.

# References

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Maintenance (changes required to run napari, tests, & CI smoothly)
- [ ] Enhancement (non-breaking improvements of an existing feature)
- [ ] Documentation (update of docstrings and deprecation information)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

cc @jni

### CODE REVIEW

**Comment by brisvag** [comment]
Not sure how useful this is, since once the PR is open it's not gonna be visible.

**Comment by jni** [comment]
@brisvag should it not be a comment then? But at least it's a semi-prominent place for contributors.

But @lucyleeow maybe the right place for this is at the bottom of the [contributing guide](https://github.com/napari/docs/blob/37ebe2dcf873d4363a43cebd13ddc5fbe825f87c/docs/developers/contributing.md), immediately after ["sharing your changes"](https://github.com/napari/docs/blob/37ebe2dcf873d4363a43cebd13ddc5fbe825f87c/docs/developers/contributing.md#sharing-your-changes)? But that is a very long guide... And our template is a long template... 😩

**Comment by brisvag** [comment]
But if it's not a comment it adds even more noise and/or people will remove it anyways... Would be great if there was a space always displayed in the PR pages where we could put this info permanently :/

A bot that answers new prs with some links would work best, but I know people don't like it cause it feels a bit cold.

**Comment by lucyleeow** [comment]
> But @lucyleeow maybe the right place for this is at the bottom of the [contributing guide](https://github.com/napari/docs/blob/37ebe2dcf873d4363a43cebd13ddc5fbe825f87c/docs/developers/contributing.md), immediately after ["sharing your changes"](https://github.com/napari/docs/blob/37ebe2dcf873d4363a43cebd13ddc5fbe825f87c/docs/developers/contributing.md#sharing-your-changes)? But that is a very long guide... And our template is a long template... weary

I can add here too, but agree that it is not that prominent so this info should not just be here.

**Comment by jni** [comment]
> What about I move it to the top of the template?

That has the potential to increase noise even more... See #6085. But it looks like @Carreau is almost ready with a PR to fix that... 😃 

Anyway, given the incoming infra fix in #6086 I think it is low risk to merge this as-is.

**Review by jni — APPROVED** [review]

**Comment by lucyleeow** [comment]
> That has the potential to increase noise even more

As did not realise you were talking about commit message noise when the comment are not removed? It would be noise as it would be an extra comment to remove?

**Comment by brisvag** [comment]
Yeah I think we're talking about different things... I refer to the noise for contributors/reviewers, not the noise in the commit messages.

**Comment by brisvag** [comment]
I guess that's still valuable, yes... In that case I would suggest to make it more concise and general (again, trying to reduce noise :P):

```md
<!--Feel free to message us in Zulip for any question or remind us to review:
https://napari.zulipchat.com/#narrow/stream/270578-reviews -->
```

**Comment by Czaki** [comment]
As I understand the core of this problem should be addressed by a community manager, whose responsibility will be not only find such PR but also contact with best core-dev

**Comment by lucyleeow** [comment]
It appears this should have been an issue. Am happy to close this in favour of a different, better change.

> As I understand the core of this problem should be addressed by a community manager, whose responsibility will be not only find such PR but also contact with best core-dev

This is a big job, and if some aspect of it could be automated, it would save community manager time to do more difficult tasks. Unless we prefer to close the '#review' channel and rely solely on community manager to do the above work for all PRs, informing contributors of existence of the review channel is at last somewhat helpful.

**Comment by lucyleeow** [comment]
> I like that a lot @Carreau

Just say the word and I am happy to close this.

> What about a bot that post messages to PRs from first time contributors like

Potentially some of the info in the PR template (namely the checklist ?) could be moved to this bot and reduce noise? see: #6092

**Comment by brisvag** [comment]
> It appears this should have been an issue.

It's ok, PRs with actuals change are often triggering better discussion than issues anyways :P

> Potentially some of the info in the PR template (namely the checklist ?) could be moved to this bot and reduce noise? see: https://github.com/napari/napari/pull/6092

+1 for the bot for me too! However (despite my noise complaints :sweat_smile:) I do think that having a reminder like that checklist in the template is useful to get higher quality PRs from the get-go.

**Comment by Czaki** [comment]
> Unless we prefer to close the '#review' channel and rely solely on community manager to do the above work for all PRs, informing contributors of existence of the review channel is at least somewhat helpful.

for me, currently, there is often a lack of proper information in the reviews channel on what is type of PR that makes me hard to determine if I am a proper person.

**Comment by lucyleeow** [comment]
Fair point and I don't think there is a perfect answer for this. I am not sure solely relying on community manager is the answer?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -37,3 +37,6 @@ Closes #
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] If I included new strings, I have used `trans.` to make them localizable.
       For more information see our [translations guide](https://napari.org/developers/translations.html).
+
+<!--Feel free to message us in Zulip for any questions or remind us to review:
```
Comment (Czaki) [inline .github/PULL_REQUEST_TEMPLATE.md:41]:
```suggestion
<!--Feel free to message us in Zulip for any questions or remind us to review if there is no reaction in 4 days:
```

Maybe something like this? To point that there is expected to wait few days?

Comment (lucyleeow) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: Yes maybe a guide on how long to wait would be useful.

Though I think I will close this in favour of the bot suggested by @Carreau. Are you supportive of such a bot: https://github.com/napari/napari/pull/6091#issuecomment-1649511666 ?

Comment (lucyleeow) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: Just waiting for any further objections

Comment (Czaki) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: I like boot too.

**Comment by psobolewskiPhD** [comment]
Re: bot -- I think it's a nice idea, especially with all the extra small things that are "normal" but not well documented like:
- editing/updating the first post of the PR and title
- the copy-edit group
- the overall review process
For what it's worth, this was once suggested by Melissa in the docs workgroup and discussed on core dev zulip. At the time it received a less than enthusiastic response, but now who knows?
At the same time, unfortunately we don't get a lot of new contributors, so...

**Comment by melissawm** [comment]
> At the same time, unfortunately we don't get a lot of new contributors, so...

We can change that 😄 Here's some wording that I find very nice - and I personally don't think it's too cold, but of course YMMV.

From [the matplotlib repo](https://github.com/matplotlib/matplotlib/pull/26383#pullrequestreview-1544287529):

```
Thank you for opening your first PR into Matplotlib!

If you have not heard from us in a week or so, please leave a new comment below 
and that should bring it to our attention. Most of our reviewers are volunteers and
sometimes things fall through the cracks.

You can also join us [on gitter](https://gitter.im/matplotlib/matplotlib) for real-time discussion.

For details on testing, writing docs, and our review process, please see [the developer guide](https://matplotlib.org/devdocs/devel/index.html)

We strive to be a welcoming and open project. Please follow our [Code of Conduct](https://github.com/matplotlib/matplotlib/blob/main/CODE_OF_CONDUCT.md?rgh-link-date=2023-07-24T21%3A15%3A18Z).
```

**Comment by brisvag** [comment]
^ love this

**Pull request closed without merging**